_sass/scaffolds.scss

```diff
@@ -270,6 +270,19 @@ li.sticky-nav{
   margin-right: 0.5em;
   flex: 5;
 }
+.project-banner-separator {
+  position: relative;
+  max-width: 100vw;
+  //left: -33px;
+  top: 0;
```

Comment:
```suggestion
  top: -3em;
```

Reply:
i still like this better than -3em.... that's my taste...